writing-style-guide.md

```diff
@@ -0,0 +1,267 @@
+# Writing Style Guide
+
+> **DRAFT**
+
+These are technical writing guidelines that should be used for Ably's developer documentation.
+
+Some of the benefits of using this style guide are:
+
+* Instructions are easier to read and understand.
+* More professional feel to documentation.
+* Documentation is easier to translate to multiple languages.
+
+As with most guidelines these are suggestions. Technical writers and contributors should always use their best judgement given the specific circumstances.
+
+This guide does not cover the details of marking up content source. See the [documentation formatting guide](https://github.com/ably/docs/blob/main/content/client-lib-development-guide/documentation-formatting-guide.textile) for the latest details on how to markup source content.
+
+## Use International English
+
+The company standard is, as with most software companies, to use International English. This is [defined](https://www.star-ts.com/about/translation-faq/what-is-international-english) as US spelling, with Americanisms removed.
+
+The industry standard dictionary is [Merriam Webster](https://www.merriam-webster.com/).
+
+## Use inclusive language
+
+Use inclusive language. Google has written extensively and thoughtfully on this subject, so there is no point in duplicating that here. See the [Google guide](https://developers.google.com/style/inclusive-documentation) for details on writing inclusive language
+
+## Empathy
+
+Empathy for developers should always be a key factor in your writing.
+
+## Use present tense
+
+Use present tense. It’s easier to read and translate.
+
+Examples:
+
+* *Future:* Command X will start the server.
+* *Present:* Command X starts the server.
+
+## Use simple language
+
+For technical writing the best approach is to be simple and direct. To achieve this objective use clear, simple, and directive language. 
+
+Avoid being overly casual in developer documentation as this can lead to a sense of lack of professionalism. However, a more casual writing style may be suitable for more relaxed pieces such as specific blog posts.
+
+Other things to avoid:
+
+* Avoid verbose writing styles. 
+* Avoid using attempts at humour in technical writing.
+
```

Comment:
I think perhaps here it is worth noting the possibly fallacy that it's ok to use imprecise language for the sake of a more informal style. Technical writing needs IMO to be unapologetically accurate and precise, and use specific terminology wherever necessary to ensure that the correct and unambiguous meaning is conveyed.

Reply:
Completely agree. I've clarified and also added your text there as I think it neatly sums up the requirement.